lib/puppet/provider/templates/webadministration/logproperties.ps1.erb

```diff
@@ -1,9 +1,44 @@
+$iis_version = '<%= Facter.value(:iis_version) %>'
+
+$W3CValue = 'W3C'
+$MaxSizeValue = 'MaxSize'
+
+# Convert the string representation of an Enum value into an Int32 safely
+Function ConvertStringTo-EnumInt32($EnumType, $Value, $ValueName) {
```

Comment:
Would rather `ConvertStringTo-EnumInt32` be `ConvertTo-EnumInt32`, it breaks PowerShell verb-noun naming conventions and doesn't add any addition info, that couldn't be better expressed in a attributed param statement:

```
function ConvertTo-EnumInt32 {
  param(
    [string]$EnumType,
    [string]$Value,
    [string]$ValueName
    )
}
```